### DISCUSSION TITLE
Python 3.12/3.13 support?

### DISCUSSION THREAD

**Discussion by i0ntempest** [open-webui/open-webui#7836] [Ideas]
# Feature Request

**Is your feature request related to a problem? Please describe.**
Can this project support Python 3.12 and 3.13? 3.11 is at "security fix" status and is only provided in source form. I forced pip to install open webui with 3.12 and it works just fine. It's just one line of change in pyproject.toml.

**Describe the solution you'd like**
Support Python 3.12 and possibly 3.13

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by i0ntempest** [comment]
https://github.com/open-webui/open-webui/pull/7850

**Comment by Kevin130703** [comment]
3.13 failed 
![Uploading image.png…]()

**Reply by i0ntempest** [reply]
You image didn't get uploaded

**Comment by astr0gator** [comment]
👀

**Comment by mslinn** [comment]
One year later, and still no support for Python 3.13

**Reply by Classic298** [reply]
We are (i think) one dependency away from python 3.13 support.

You **could help** with achieving this.

Yeah it's just one last dependency that's still on python 3.12. Then we could have python 3.13
